doc/api/fs.md

```diff
@@ -5367,6 +5367,20 @@ The following constants are exported by `fs.constants`.
 
 Not every constant will be available on every operating system.
 
+To use more than one constant, use the bitwise OR `|` operator.
+
+Example:
+
+```js
+const fs = require('fs');
+
+const flags = fs.constants.O_RDWR | fs.constants.O_CREAT | fs.constants.O_EXCL;
+
+fs.open('<file>', flags, (err, fd) => {
```

Comment:
Purely stylistic nit, feel free to ignore if you disagree

```suggestion
const fs = require('fs');

const {
  O_RDWR,
  O_CREAT,
  O_EXCL
} = fs.constants

fs.open('/path/to/my/file', O_RDWR | O_CREAT | O_EXCL, (err, fd) => {
```

Reply:
The destructuring does read better 👍